### ISSUE TITLE
Unable To Send Direct Message.

### ISSUE THREAD

**Issue by teezzan** [FeedHive/twitter-api-client#37]
Thanks for your awesome work. 

I have been unable to send direct messages while following the [reference document](https://github.com/Silind/twitter-api-client/blob/master/REFERENCES.md#twitterclientdirectmessagesdirectmessageseventsnewmessagecreateparameters).

Whenever I run 
```javascript
let parameters ={id:"1322838158258720769", data:"helloworld"}
twitterClient.directMessages.directMessagesEventsNewMessageCreate(parameters)
.catch(err => { console.log(err) })
```
the library keeps returning `{ statusCode: 415, data: '' }`.

What am I doing wrong? 

My current version is `"twitter-api-client": "^1.2.0"`

Thank You Very Much

**Comment by Silind** [comment]
Hi, @teezzan.
Thanks for creating this issue.

Your code looks correct, so it's likely to be a bug.
I'll look into this asap.

**Comment by teezzan** [comment]
Thank you very much.

**Comment by lcswillems** [comment]
It is because twitter-api-client doesn't send the correct request. I opened an issue for this: https://github.com/FeedHive/twitter-api-client/issues/45

**Comment by teezzan** [comment]
Thanks. Hopefully a fix will be available soon.

**Comment by SimonHoiberg** [comment]
Fixed in version 1.2.9
